### ISSUE TITLE
Remove my vote

### ISSUE THREAD

**Issue by iSapozhnik** [AvdLee/Roadmap#18]
Is there a way to remove my vote?
For instance I voted for one feature but then I. saw another one which I actually like more and don't want the first one anymore. I would expect that clicking on the feature I voted before will reset it back.

Thanks

**Comment by jordibruin** [comment]
They're not an easy way to do that with countapi.xyz (although there is a way). Lets investigate!

**Comment by iSapozhnik** [comment]
Yeah I see. Just sniffed the traffic to check what does it return (like if there is an ID we can store locally) but no - nothing. 🤔 I'm afraid with the countAPI it's not even possible.

**Comment by jordibruin** [comment]
It is possible with the /set/ endpoint

/set/:namespace?/:key?value=:value

Set the value of a key. Optionally specify the namespace. The key must be created with enable_reset set to 1 (true).

This endpoint will return the previous value before the assignation.

We would have to save the current votes and then set it to one below that.

**Comment by vdhamer** [comment]
> It is possible with the /set/ endpoint
> /set/:namespace?/:key?value=:value
> Set the value of a key. Optionally specify the namespace. The key must be created with enable_reset set to 1 (true).
> This endpoint will return the previous value before the assignation.
> We would have to save the current votes and then set it to one below that.

The /set/ endpoint would do the trick.
If you can get /create to work with parameters.
But I tried, and even the examples in the `create` API documentation don't work.
api.countapi.xyz got sold, and the new owner either accidentally broke it, or doesn't like fuss associated with possible data loss??
See [transfer in 2021](https://mlomb.me/projects).

Alternatives:
* contact new `counterapi` owner (might just be a bug) to ensure the API docs for /create and the implementation match
* contact original author of `counterapi` to somehow ensure the problem gets resolved. He will have an easier time to figure out what happened and how to get it fixed.
* with `/create` you _can_ create default counters that can be incremented and decremented by 1. But, because you can't override any default parameters, you end up in namespace 'default' (hmmm) with key=UUID (hmmm). Pretty ugly, but gives you as many keys as you like. So the trick here is create counters using `/create` instead of `/hit`
* tell the user that voting cannot be undone (I did) and somehow add an .alert dialog to force a confirmation (I tried using a wrapper around the default voting, but it doesn't work yet)

**Comment by mlomb** [comment]
Hi, original author of CountAPI here.
Yes, the only way to reduce a counter is using `/set` with `enable_reset=1`. If it doesn't work it must be a bug 😢. I think APILayer hasn't changed the code at all for now.

**Comment by jordibruin** [comment]
@mlomb thanks so much for creating countapi! 🥰
